reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md

```diff
@@ -373,15 +376,17 @@ Accept wildcard characters: False
 
 ### -ComputerName
 
-Specifies a remote computer. If a **ComputerName** is not specified, `Get-EventLog` uses the local
-computer.
+This parameter specifies a remote computer's NetBIOS name, Internet Protocol (IP) address, or a
+fully qualified domain name (FQDN).
 
-Type the NetBIOS name, an Internet Protocol (IP) address, or a fully qualified domain name (FQDN)
-of a remote computer. To specify the local computer, type the computer name, a dot (`.`), or
-localhost.
+If the **ComputerName** parameter isn't specified, `Get-EventLog` uses the default, which is the
```

Comment:
Change this to:

> If the **ComputerName** parameter isn't specified, `Get-EventLog` defaults to the local
> computer. The parameter also accepts a dot (`.`) to specify the local computer.

Don't mention "localhost"

Reply:
Resolved, thanks for the feedback.